src/04-arrays.js

```diff
@@ -135,7 +155,12 @@ function removeElementFromFrontOfArray(array) {}
  *  getMiddleElement([ 10, null, "30" ]);
  *  //> null
  */
-function getMiddleElement(array) {}
+function getMiddleElement(array) {
+  if (array.length === 1){
```

Comment:
Take another look here.  How would we write some code to find the middle element of any array?  What is our array has a `length` of `20`?  Or `15`?.